docker/README.md

```diff
@@ -1,6 +1,6 @@
 # Build docker image
```

Comment:
`docker` dir should go away. You can leave `config_templage.yaml` for now (I will move it later) but the content of `README.md` should be migrated to the root one (with proper modifications, of course). Or to some `DOCKER.md` if you don't want to pollute main file.

Reply:
moved docker readme here https://github.com/temporalio/docker-builds/pull/17
added comment about docker images https://github.com/temporalio/temporal/pull/2456/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R32